site-src/implementations.md

```diff
@@ -354,26 +356,37 @@ Check out the [Gateway API task][istio-2] for more information about the Gateway
+[![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v1.1.0-Kong%20Ingress%20Controller-green)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.1.0/kong-kubernetes-ingress-controller)
 
 [Kong][kong] is an open source API Gateway built for hybrid and multi-cloud environments.
 
-Kong supports Gateway API in the [Kong Kubernetes Ingress Controller (KIC)][kic], see the [Gateway API Guide][kong-gw-guide] for usage information.
+The [Kong Kubernetes Ingress Controller (KIC)][kic] can be used to configure unmanaged Gateways. See the [Gateway API Guide][kong-gw-guide] for usage information.. See the [Gateway API Guide][kong-gw-guide] for usage information.
 
-Kong also supports Gateway API in the [Kong Gateway Operator][kgo].
-
-For help and support with Kong's implementations please feel free to [create an issue][kong-issue-new] or a [discussion][kong-disc-new]. You can also ask for help in the [#kong channel on Kubernetes slack][kong-slack].
+For help and support with Kong Kubernetes Ingress Controller please feel free to [create an issue][kic-issue-new] or a [discussion][kic-disc-new]. You can also ask for help in the [#kong channel on Kubernetes slack][kong-slack].
 
 [kong]:https://konghq.com
 [kic]:https://github.com/kong/kubernetes-ingress-controller
 [kong-gw-guide]:https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/using-gateway-api/
-[kgo]:https://docs.konghq.com/gateway-operator/latest/
-[kong-issue-new]:https://github.com/Kong/kubernetes-ingress-controller/issues/new
-[kong-disc-new]:https://github.com/Kong/kubernetes-ingress-controller/discussions/new
+[kic-issue-new]:https://github.com/Kong/kubernetes-ingress-controller/issues/new
+[kic-disc-new]:https://github.com/Kong/kubernetes-ingress-controller/discussions/new
 [kong-slack]:https://kubernetes.slack.com/archives/CDCA87FRD
 
+### Kong Gateway Operator
+
+[![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v1.2.0-Kong%20Gateway%20Operator-orange)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.2.0/kong-gateway-operator)
+
+[Kong][kong] is an open source API Gateway built for hybrid and multi-cloud environments.
+
+The [Kong Gateway operator (KGO)][kgo] can be used to configure managed Gateways.
+
+For help and support with Kong Gateway operator please feel free to [create an issue][kgo-issue-new] or a [discussion][kgo-disc-new]. You can also ask for help in the [#kong channel on Kubernetes slack][kong-slack].
+
+[kgo]:https://docs.konghq.com/gateway-operator/latest/
+[kgo-issue-new]:https://github.com/Kong/kubernetes-gateway-operator/issues/new
+[kgo-disc-new]:https://github.com/Kong/kubernetes-ingress-controller/discussions/new
```

Comment:
Is this link meant to point to the ingress controller?

Reply:
good catch, the link was actually wrong. Thank you!